src/utils/string.formatter.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * String formatting functions
+ */
+const REGEX_MATCH_NON_PRINTABLE_CHARS: RegExp =
+    /[^ÀÁÂÄÈÉÊËÌÍÎÏÒÓÔÖÙÚÛÜẀẂŴẄỲÝŶŸàáâäèéêëìíîïòóôöùúûüẁẃŵẅỳýŷÿABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789&@£$€¥.,:;–\-'''()\[\]{}<>!""»«"\?\/\\*=#%\+ \r\n]/g;
```

Comment:
I think sonar is complaining because {} aren't escaped.
Try \{\} and put a \ in front of +

Reply:
I cannot see this on Sonar.

Reply:
Lint complains when you do escape then  - what advice does Sonar give.

Reply:
http://code-analysis.platform.aws.chdev.org:9000/dashboard?id=uk.gov.companieshouse%3Astrike-off-objections-web&pullRequest=137

This is the sonar report, essentially needs less +*{ characters

Reply:
May also be worth simplifying A-Za-z0-9 for easier readability.
You seem to have char code 39 (') 3 times, you only need it once, same for char code 34 (")

Reply:
Simplifyed. The repeated ' is as miscopy it should be ` changed this.